pom.xml

```diff
@@ -27,40 +27,38 @@
         </dependency>
     </dependencies>
     <build>
-        <plugins>
```

Comment:
revert all changes in **pom.xml**

Reply:
My project does not build if I return the default pom